docs/README.md

```diff
@@ -1,11 +1,7 @@
-# ZOSOpenTools docs
+## Welcome to z/OS Open Tools
 
-To make changes to the docs, please:
```

Comment:
i agree this part shouldn't be on the front page, but did it completely disappear?

Reply:
Thanks, I added a reference to it in the latest commit:

<img width="1196" alt="Screen Shot 2022-08-16 at 12 03 15 PM" src="https://user-images.githubusercontent.com/39890068/184926403-e3cd9a4f-983a-4ecb-b005-242337758dca.png">